src/angular/header-lean/header.md

```diff
@@ -0,0 +1,88 @@
+The header will appear at the top of the screen in a fixed position, and provide a container
```

Comment:
This file should probably called header-lean.md. At the moment its not shown in the showcase.